t/01-compile-check-all.t

```diff
@@ -20,12 +20,19 @@ use FindBin;
 unshift @INC, "$FindBin::Bin/lib", "$FindBin::Bin/../lib";
 
 use Test::Strict;
+use Mojo::File;
 
 $Test::Strict::TEST_SYNTAX   = 1;
 $Test::Strict::TEST_STRICT   = 1;
 $Test::Strict::TEST_WARNINGS = 1;
-$Test::Strict::TEST_SKIP     = [
+
+$Test::Strict::TEST_SKIP = [
     # skip test module which would require test API from os-autoinst to be present
-    't/data/openqa/share/tests/opensuse/tests/installation/installer_timezone.pm'
+    't/data/openqa/share/tests/opensuse/tests/installation/installer_timezone.pm',
+    # problem: when tidy test fails - it leaves .tdy files, so we should ignore them
```

Comment:
I think you fixed a problem I never understood but have seen in some test runs, great!